src/propTypes.js

```diff
@@ -3,6 +3,6 @@ const PropTypes = require("prop-types");
 module.exports = {
   refType: PropTypes.oneOfType([
     PropTypes.func,
-    PropTypes.shape({ current: PropTypes.elementType })
+    PropTypes.shape({ current: PropTypes.object })
```

Comment:
`elementType` refers to a React element, but here we may have a _DOM_ element.

I initially tried to fix this using `PropTypes.oneOfType([PropTypes.elementType, PropTypes.instanceOf(Element)])`, but this broke when the code was ran on the server, as `Element` does not exist as a global there.